src/main/java/org/embulk/output/SnowflakeOutputPlugin.java

```diff
@@ -175,6 +191,8 @@ protected void logConnectionProperties(String url, Properties props) {
         maskedProps.setProperty(key, "***");
       } else if (key.equals("proxyPassword")) {
         maskedProps.setProperty(key, "***");
+      } else if (key.equals("privateKey")) {
+        maskedProps.setProperty(key, "***");
```

Comment:
idea: to use `props.keySet()`

Reply:
thank you for suggesting. i fixed it.
https://github.com/trocco-io/embulk-output-snowflake/pull/65/files#diff-05c2b1752e71c7d5d413d78a81b5e80ec268850049f70e31f071c3dae13f1cd1R189-R190